Makefile

```diff
@@ -29,7 +30,7 @@ lint:
 
 load:
 	SERVER_URL=$(STAGE_SERVER_URL) ENDPOINT_URL=$(STAGE_ENDPOINT_URL) \
-	  docker-compose \
+	  $(DOCKER_COMPOSE) \
```

Comment:
Should we re-use this in the `load-clean` as well?

Reply:
I thought I pushed that already, yes!